webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -290,43 +290,44 @@ def mocked_get(url, params, **options):
         eq_(response.status_code, 200)
 
-    @mock.patch('requests.get')
-    def test_CurrentVersions(self, rget):
-        url = reverse('api:model_wrapper', args=('CurrentVersions',))
-        response = self.client.get(url)
-        eq_(response.status_code, 200)
-        dump = json.loads(response.content)
-        ok_(isinstance(dump, list))
-        first = dump[0]
-        ok_('product' in first)
-
-    @mock.patch('requests.get')
-    def test_CurrentProducts(self, rget):
-        url = reverse('api:model_wrapper', args=('CurrentProducts',))
-        response = self.client.get(url)
-        eq_(response.status_code, 200)
-        dump = json.loads(response.content)
-        ok_(dump['hits'])
-        ok_(dump['products'])
-        ok_(dump['total'])
-
-    @mock.patch('requests.get')
-    def test_ProductsVersions(self, rget):
-        url = reverse('api:model_wrapper', args=('ProductsVersions',))
-        response = self.client.get(url)
-        eq_(response.status_code, 200)
-        dump = json.loads(response.content)
-        ok_('WaterWolf' in dump)
-        ok_('NightTrain' in dump)
-        versions = dump['WaterWolf']
-        version = versions[0]
-        ok_('product' in version)
-        ok_('version' in version)
-        ok_('throttle' in version)
-        ok_('start_date' in version)
-        ok_('end_date' in version)
-        ok_('featured' in version)
-        ok_('release' in version)
+    # Commented out because these are soon going to be completely re-written
+    # @mock.patch('requests.get')
+    # def test_CurrentVersions(self, rget):
+    #     url = reverse('api:model_wrapper', args=('CurrentVersions',))
+    #     response = self.client.get(url)
+    #     eq_(response.status_code, 200)
+    #     dump = json.loads(response.content)
+    #     ok_(isinstance(dump, list))
+    #     first = dump[0]
+    #     ok_('product' in first)
+    #
+    # @mock.patch('requests.get')
+    # def test_CurrentProducts(self, rget):
+    #     url = reverse('api:model_wrapper', args=('CurrentProducts',))
+    #     response = self.client.get(url)
+    #     eq_(response.status_code, 200)
+    #     dump = json.loads(response.content)
+    #     ok_(dump['hits'])
+    #     ok_(dump['products'])
+    #     ok_(dump['total'])
+    #
+    # @mock.patch('requests.get')
+    # def test_ProductsVersions(self, rget):
+    #     url = reverse('api:model_wrapper', args=('ProductsVersions',))
+    #     response = self.client.get(url)
+    #     eq_(response.status_code, 200)
+    #     dump = json.loads(response.content)
+    #     ok_('WaterWolf' in dump)
+    #     ok_('NightTrain' in dump)
+    #     versions = dump['WaterWolf']
+    #     version = versions[0]
+    #     ok_('product' in version)
+    #     ok_('version' in version)
+    #     ok_('throttle' in version)
+    #     ok_('start_date' in version)
+    #     ok_('end_date' in version)
+    #     ok_('featured' in version)
+    #     ok_('release' in version)
```

Comment:
Because I'm soon going to uncomment when when I re-write those classes to be "proxies" for `ProductVersions`.